controllers/default.py

```diff
@@ -71,9 +76,11 @@ def api():
     return Collection(db).process(request,response,rules)
 
 def register():
+    auth.settings.register_next = 'confirm.html'
     response.title = 'iDibo Registration'
-    return dict()
+    return dict(form = auth.register())
```

Comment:
Please follow PEP8 "Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value." https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements So, the spaces should be line in line 44 above.